SalesforceSDK/Salesforce.SDK.SmartSync/Manager/SyncManager.cs

```diff
@@ -569,11 +469,11 @@ private HashSet<string> ToSet(JArray jsonArray)
             }
         }
 
-        private string AddFilterForReSync(string query, long maxTimeStamp)
+        public static string AddFilterForReSync(string query, long maxTimeStamp)
```

Comment:
On the java side, I moved that function to the soql sync target - since it's very soql specific